### DISCUSSION TITLE
how to update Details!

### DISCUSSION THREAD

**Discussion by lacerated** [layday/instawow#122] [General]
Is there a way to update Details addon from github?

**Comment by layday** [comment]
Afraid not, Details is only available from Curse and Wago.

**Reply by lacerated** [reply]
and i cant update details using instawow rom curse or wago?

**Reply by layday** [reply]
You can generate a key for CurseForge by following the instructions on the options pane in the GUI or by running `instawow configure access_tokens.cfcore` in the CLI, but I believe Details forbids distribution from the public API.  Wago is not supported.
